tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1683,6 +1686,41 @@ def unique_id(self):
+        if not AutoTuner.get().is_tuning_mode or not AutoTuner.get(
+        )._is_preparation:
+            return
+        if not hasattr(torch.ops.trtllm, "preallocate_nccl_window_buffer"):
+            return
+        if input_tensor.numel() == 0 or input_tensor.size(0) == 0:
+            return
+        if hasattr(torch.cuda, "is_current_stream_capturing"):
+            try:
+                if torch.cuda.is_current_stream_capturing():
+                    return
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)